src/Transactions/TransactionManager.php

```diff
@@ -32,7 +32,7 @@ public function createDbDriver()
         );
     }
 
-    protected function getContainer()
+    public function getContainer()
```

Comment:
Thanks for this :) It was recently added so I was not aware of it. This change is already in master